dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -995,9 +1002,10 @@ protected String getCommitID(String repositoryId, Version version) {
 
             for (GHRef ref : refs) {
                 String reference = StringUtils.removePattern(ref.getRef(), "refs/.+?/");
-                if (reference.equals(version.getReference())) {
-                    return ref.getObject().getSha();
-                }
+                // When a user creates an annotated tag, the object type will be a tag. Otherwise, it's probably of type commit?
```

Comment:
might make more sense to move the comment to the method

Reply:
Agreed with Gary's comment, I think this belongs with the method.